src/model/extension.ts

```diff
@@ -135,6 +136,45 @@ export class DataFilterExtension extends BaseExtensionModel {
   }
 }
 
+export class PathStyleExtension extends BaseExtensionModel {
+  static extensionType = "path-style";
+
+  extensionInstance: _PathStyleExtension;
+
+  constructor(
+    model: WidgetModel,
+    layerModel: BaseLayerModel,
+    updateStateCallback: () => void,
+  ) {
+    super(model, updateStateCallback);
+
+    const dash = this.model.get("dash");
+    const highPrecisionDash = this.model.get("high_precision_dash");
+    const offset = this.model.get("offset");
+    this.extensionInstance = new _PathStyleExtension({ dash });
+
+    // Properties added by the extension onto the layer
+    layerModel.initRegularAttribute("get_dash_array", "getDashArray");
+    layerModel.initRegularAttribute("dash_justified", "dashJustified");
+    layerModel.initRegularAttribute("get_offset", "getOffset");
+    layerModel.initRegularAttribute(
+      "dash_gap_pickle",
+      "dashGapPickle",
+    );
+    layerModel.initVectorizedAccessor("get_filter_value", "getFilterValue");
```

Comment:
Oh actually you want to delete `getFilterValue`; that only applied to the data filter extension

Reply:
Removed! Was an artefact from the copy!